performance/scaling_data_ingestion.rst

```diff
@@ -5,8 +5,6 @@ Scaling Out Data Ingestion
 
 Citus lets you scale out data ingestion to very high rates, but there are several trade-offs to consider in terms of the throughput, durability, consistency and latency. In this section, we discuss several approaches to data ingestion and give examples of how to use them.
 
-The best method to distribute tables and ingest your data depends on your use case requirements. Citus supports two distribution methods: append and hash; and the data ingestion methods differ between them. You can visit the :ref:`working_with_distributed_tables` section to learn about the tradeoffs associated with each distribution method.
-
 Hash Distributed Tables
```

Comment:
Maybe let's remove this top-level heading on hash-distribution? We used to have another on 'Append', but now that its removed, no reason to keep this.